content/manuals/docker-hub/service-accounts.md

```diff
@@ -7,62 +7,33 @@ weight: 50
 
 {{< include "new-plans.md" >}}
 
-> [!NOTE]
+> [!IMPORTANT]
 >
-> Service accounts require a
-> [Docker Team, or Business subscription](../subscription/_index.md).
+> As of December 10, 2024, service accounts are no longer available. Existing
+> Service Account agreements will be honored until their current term expires,
+> but new purchases or renewals of service accounts no longer available and
+> customers must renew under a new subscription plan.
 >
-> Docker recommends that you use organization access tokens instead of service
-> accounts. For more details, see [Organization access
-> tokens](/security/for-admins/access-tokens/).
+> Docker recommends transitioning to [Organization Access Tokens
+> (OATs)](../security/for-admins/access-tokens.md), which can provide similar
+> functionality.
 
 A service account is a Docker ID used for automated management of container images or containerized applications. Service accounts are typically used in automated workflows, and don't share Docker IDs with the members in the organization. Common use cases for service accounts include mirroring content on Docker Hub, or tying in image pulls from your CI/CD process.
 
 > [!NOTE]
 >
 > All paid Docker subscriptions include up to 5000 pulls per day per authenticated user. If you require a higher number of pulls, you can purchase an Enhanced Service Account add-on.
```

Comment:
Nit/thought: Is this disclaimer fine as is, or is it confusing to read "you can purchase an Enhanced Service Account add-on." in a disclaimer when there's the "no longer available" disclaimer above?